src/components/common/badge/RelationshipBadge.jsx

```diff
@@ -0,0 +1,24 @@
+//관계 뱃지 컴포넌트
+
+import { cva } from 'class-variance-authority';
+import { cn } from '@/utils/style';
+
+const relationshipBadgeStyle = cva(
+  'inline-flex items-center justify-center rounded px-2 py-0.5 font-14-regular h-[20px]',
+  {
+    variants: {
+      type: {
+        친구: 'bg-blue-100 text-blue-600',
+        가족: 'bg-green-100 text-green-600',
+        동료: 'bg-purple-100 text-purple-600',
+        지인: 'bg-orange-100 text-orange-600',
+      },
+    },
+  }
+);
+
+const RelationshipBadge = ({ type = '지인' }) => {
+  return <span className={cn(relationshipBadgeStyle({ type }))}>{type}</span>;
+};
+
```

Comment:
propTypes가 빠졌습니다!!
제가 저번에 남긴 리뷰 위치가 애매했네요...🥲
propTypes는 이전처럼 두고 prop의 기본값을 제거하면 될 것 같습니다!
-> propTypes로 정의한 type의 값이 isRequired여서 기본 값을 생략해도 됩니다!

```suggestion
const RelationshipBadge = ({ type }) => {
  return <span className={cn(relationshipBadgeStyle({ type }))}>{type}</span>;
};

RelationshipBadge.propTypes = {
  type: PropTypes.oneOf(['친구', '가족', '동료', '지인']).isRequired,
  }
```